.typedoc/typedoc.config.mjs

```diff
@@ -0,0 +1,117 @@
+/** @type {Partial<import("typedoc").TypeDocOptions>} */
+const config = {
+  tsconfig: "./tsconfig.sdk-docs.json",
+  plugin: [
+    "typedoc-plugin-missing-exports",
+    "typedoc-plugin-markdown",
+    "typedoc-plugin-frontmatter",
+    "typedoc-plugin-mdn-links",
+  ],
+  entryPoints: ["../resources/embedding-sdk/dist/index.d.ts"],
+  router: "structure",
+  internalModule: "internal",
+  outputs: [
+    {
+      name: "html",
+      path: "../docs/embedding/sdk/generated/html",
+      options: {
+        favicon: "./resources/frontend_client/favicon.ico",
+        customJs: "./.typedoc/page-custom-logic.js",
+        customCss: "./.typedoc/page-custom-styles.css",
+        hideGenerator: true,
+        collapseInternalModule: true,
+        visibilityFilters: {},
+        frontmatterGlobals: {
+          layout: "docs-api",
+        },
+        pretty: true,
+        customFooterHtml:
+          '<script type="text/javascript" src="/gdpr-cookie-notice/dist/script.js"></script>' +
+          '<script type="text/javascript" src="/js/cookie-consent.js"></script>' +
+          '<link href="/gdpr-cookie-notice/dist/style.css" rel="stylesheet" />' +
+          '<link href="/css/gdpr.css" rel="stylesheet" />',
```

Comment:
We have to display a custom GDPR banner on generated pages, so here it is

Reply:
Yes, we have it there. On the Embedding SDK API pages we use local/session storages, so ideally we should have it there. 

Plus this banner comes with analytics scripts, so we must have it there.